modules/transactions/src/test/java/org/apache/ignite/internal/tx/storage/state/TxStateStorageAbstractTest.java

```diff
@@ -111,7 +111,7 @@ public void testCas() throws Exception {
 
             UUID txId = UUID.randomUUID();
 
-            TxMeta txMeta0 = new TxMeta(TxState.PENDING, new ArrayList<>(), generateTimestamp(txId));
+            TxMeta txMeta0 = new TxMeta(null, new ArrayList<>(), generateTimestamp(txId));
```

Comment:
There's no sense in TxMeta(null). TxMeta either null itself or has not null state internally. So that, such TxMeta(null ... steps should be removed.

Reply:
fixed